app/js/FullPage.js

```diff
@@ -41,26 +41,26 @@ export class FullPage {
     let that = this;
 
     this.fpOnLeave.push(function(index, nextIndex, direction) {
-      new Decorations().closeMenuIfOpen();
-    })
+      new Menu().closeMenuIfOpen();
+    });
 
     this.fpOnLeave.push(function(index, nextIndex, direction) {
       if (!that.shown && nextIndex == 7) {
         that.shown = true;
         setTimeout(() => $('.featureIcon:first').trigger('click'), 1000);
       }
-    })
+    });
 
     // Hide menu logo on center page
     this.fpOnLeave.push(function(index, nextIndex, direction) {
-      if (nextIndex === that.centerpage) {
+      if (nextIndex === that.centerPage) {
         $('.navigate-arrows').fadeIn(350);
         $('#menu-logo').fadeOut(350);
       } else {
         $('.navigate-arrows').fadeOut(350);
         $('#menu-logo').fadeIn(350);
       }
-    })
+    });
```

Comment:
There are some situations that we can avoid if we stop using elses. 
  - The first one reduces the nested code. With one only if it is not that easy to see the improvement, but in more complex situations it can improve readability.

For example, let's pretend that we have a huge piece of code that should run only if X = true. Instead do

```javascript
if (x) {
   code
   more code
   a lot of code
}
```
if we do

```javascript
if (!x) {
   return;
}

   code
   more code
   a lot of code
```

  - It is not this case, but in cases when we need to return something with the functions this can be helpful to avoid ambiguous situations.